### PR TITLE
feat : Edit Profile Page

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added Edit User Profile functionality
   - Change Bio
   - Change Website Link
   - Change Profile Picture

### CODE REVIEW

File: src/backend/db/users.js [hunk 1]
```diff
@@ -7,11 +7,28 @@ import { formatDate } from '../utils/authUtils';
...
-    _id: uuid(),
+    _id: '054c1f2b-dcf8-439b-86e7-820d4ac416fc',
     firstName: 'Adarsh',
     lastName: 'Balika',
     username: 'adarshbalak',
     password: 'adarshBalaki123',
+    bio: 'Aspring FullStack Developer',
+    website: 'https://adarshbalak.netlify.app',
+    avatar:
+      'https://res.cloudinary.com/anujy0510/image/upload/v1652788469/portrait-young-indian-top-manager-t-shirt-tie-crossed-arms-smiling-white-isolated-wall_496169-1513_ac8h4f.jpg',
```
Comment (Shah-Arsalan) [inline src/backend/db/users.js:18]: Nice that you are using cloudinary

File: src/pages/Profile/components/EditProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  p: 4,
+};
+
+export const EditProfile = ({ handleClose, open }) => {
+  const { user, token } = useSelector((state) => state.auth);
+  const dispatch = useDispatch();
+  const [bio, setBio] = useState(user.bio);
+  const [website, setWebsite] = useState(user.website);
+  const [avatar, setAvatar] = useState(user.avatar);
+  console.log(user);
```
Comment (Shah-Arsalan) [inline src/pages/Profile/components/EditProfile.jsx]: remove console logs

**Review by Shah-Arsalan — APPROVED** [review]
the code is clean and modular . You can merge

**Review by shrey27 — APPROVED** [review]
Features are working as expected
Code is well structured
PR is good to merge

File: src/pages/Profile/components/EditProfile.jsx (hunk 2 above)
Comment (anuj-kumary) [reply src/pages/Profile/components/EditProfile.jsx]: Done

**Pull request merged**